extras/docker/fromsource/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # set author and base
 FROM fedora
-MAINTAINER Luis Pabón <lpabon@redhat.com>
+MAINTAINER Michael Adam <obnox@redhat.com>
```

Comment:
What about using "Heketi Developers" and the mailinglist for contact details? I'v done that in #1202 as well.

Reply:
Done, thanks!